packages/gatsby-theme-learning/src/components/sidebar-topic-status.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from 'react';
+import { CheckActiveIcon, CircleIcon } from '@commercetools-uikit/icons';
+import { useAuth0 } from '@auth0/auth0-react';
+import {
+  useFetchCourseDetails,
+  getTopicStatusByPageTitle,
+} from '../hooks/use-course-details';
+import ConfigContext, {
+  isFeatureEnabled,
+  EFeatureFlag,
+} from './config-context';
+
+type StatusIndicatorProps = {
+  status?: string;
+};
+
+export const StatusIndicator = (props: StatusIndicatorProps) =>
+  props.status && props.status === 'completed' ? (
+    <CheckActiveIcon color="primary" size="medium" />
+  ) : (
+    <CircleIcon color="neutral60" size="medium" />
+  );
+
+type PageTopicStatusProps = {
+  courseId: number;
+  pageTitle: string;
+};
+
+const SidebarTopicStatus = (props: PageTopicStatusProps) => {
+  const { isAuthenticated } = useAuth0();
+  const { data } = useFetchCourseDetails(props.courseId);
+  const { features } = useContext(ConfigContext);
```

Comment:
It somehow feels wrong to me that it's necessary to pull the features from a hook to pass them in although the Config Context got them as props when initialized in gatsby-ssr   - but I am really not sure and have not fully read the code in detail. 

BTW tobias maintains this helper lib which is how we do it in the MC:  https://github.com/tdeekens/flopflip  (backed by a cloud service to be able to dynamically toggle per user).  Its solving problems of big apps, not really ours yet but has a lot of helpers.

Reply:
The `isFeatureEnabled` utility function is trivial and very possibly not needed, indeed I could simply `features.includes('feature-name')` and save the hassle of passing features to another helper function.

Reply:
A helper function is mandatory in my opinion,  down the line feature flagging is per user or group of users (e.g. commercetools users only etc).   It's a black box for the individual component why a feature is off or on now.

Reply:
if you feel we should look into the `flipflop` lib, we can do it but I don't think it's worth doing it now. We could create an enhancement ticket to make feature flagging more powerful in the future

Reply:
I think your simple function call is enough for what we need now.  I just referenced flipflop to you as an inspiration of how a mature set of helpter API for feature flags looks like .